src/app/store/defaultState.js

```diff
@@ -133,6 +133,7 @@ const defaultState = {
             }
         ],
         recentFiles: [],
+        grblExists: true,
```

Comment:
Why is this being stored in store and not redux?  It doesn't need to persist since it's a state check on a per connection basis, not a setting that needs to be saved on the hard drive.